src/main/java/com/sourcegraph/website/SearchActionBase.java

```diff
@@ -103,7 +108,8 @@ private String getSelectedText(@Nullable Project project) {
       return null;
     }
 
-    Editor editor = FileEditorManager.getInstance(project).getSelectedTextEditor();
+    Editor editor =
+        Arrays.stream(CodyEditorUtil.getSelectedEditors(project)).findFirst().orElse(null);
```

Comment:
This line is happening a *lot*, can we put it into `CodyEditorUtil` and be done with it?

Reply:
Done :)